src/main/java/com/algorand/algosdk/v2/client/common/Settings.java

```diff
@@ -0,0 +1,6 @@
+package com.algorand.algosdk.v2.client.common;
+
+public class Settings {
+
+	public static final String DateFormat = "yyyy-MM-dd'T'h:m:ss'Z'";
```

Comment:
Does this mean "Date" objects are always treated as UTC?

Reply:
Fixing to UTC simplifies things for sure. 
What is the functional description for this?